docs/sections/user_guide/yaml/components/make_hgrid.rst

```diff
@@ -0,0 +1,241 @@
+.. _make_hgrid_yaml:
+
+make_hgrid
+==========
+
+Structured YAML to run :ufs-utils:`make_hgrid<make-hgrid>` is validated by JSON Schema and requires the ``make_hgrid:`` block, described below. If ``make_hgrid`` is to be run via a batch system, the ``platform:`` block, described :ref:`here <platform_yaml>`, is also required.
+
+YAML keys and their descriptions are taken largely verbatim from the ``make_hgrid`` tool source code, available `here <https://github.com/NOAA-GFDL/FRE-NCtools/blob/65c34e76cfa2ff567f70da83f44cd6f991315f08/tools/make_hgrid/make_hgrid.c>`_.
```

Comment:
Is that very-specific URL preferable to the simpler https://github.com/NOAA-GFDL/FRE-NCtools/blob/master/tools/make_hgrid/make_hgrid.c ? Maybe so, if it somehow ties the docs to the specific version of `make_hgrid` being used by... well, SRW? This leads into the fraught topic of versioning drivers, docs, etc. based on the versions of apps and components. Maybe it would be better to used the unversioned URL until we know more about version-pinning requirements?

Reply:
Good point, thats safer for sure. As long as we're just doing references and not pulling full text, this should be ok